src/Command/ConfigShow.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Command;
+
+use Magento\MagentoCloud\Command\ConfigShow\Renderer as ConfigRenderer;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * CLI command to friendly display the encoded cloud configuration environment variables
+ */
+class ConfigShow extends Command
+{
+    const NAME = 'config:show';
```

Comment:
We have `config:dump` CLI command in ece-tools, where under the `config` we mean Magento configurations (ok, part of Magento configurations)
This command displays absolutely different configurations. So, just to avoid confusion, I would recommend to rename it.

Reply:
@NadiyaS I'm fine with that. It can be something like `cloud:config:show` or similar